esp-box/esp-box.c

```diff
@@ -370,3 +371,22 @@ bool bsp_button_get(const bsp_button_t btn)
 {
     return !(bool)gpio_get_level(btn);
 }
+
+bool bsp_touch_button_get(void)
```

Comment:
I don't like this new function. I rather to see add `BSP_BUTTON_MAIN` into enum `bsp_button_t`. And then handle it in `bsp_button_get` (in condition).